<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>611</width>
     <height>331</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QListWidget" name="listWidget">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>120</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="currentRow">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>用户设置</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>服务器设置</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>sip协议设置</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>媒体设置</string>
     </property>
    </item>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="pageuser">
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>451</width>
        <height>321</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>用户设置</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="username">
           <property name="text">
            <string>用户名</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="usernameLineEdit">
           <property name="echoMode">
            <enum>QLineEdit::Normal</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="passwd">
           <property name="text">
            <string>密码</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="passwdLineEdit">
           <property name="echoMode">
            <enum>QLineEdit::PasswordEchoOnEdit</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="iPLabel">
           <property name="text">
            <string>服务器IP</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="serverLineEdit"/>
         </item>
         <item row="4" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>400</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="pageserver">
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>461</width>
        <height>321</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>服务器设置</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>sip server</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="sipServerIP1"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>sip server 2</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="sipServerIP2"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="pagesip">
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>461</width>
        <height>321</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>SIP协议设置</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>杂项设置</string>
         </property>
         <widget class="QWidget" name="formLayoutWidget_3">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>411</width>
            <height>81</height>
           </rect>
          </property>
          <layout class="QFormLayout" name="formLayout_4">
           <item row="0" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>外出代理：</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>转移url：</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="outboundLineEdit"/>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="transferLineEdit"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>DTMF模式：</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="DTMFcomboBox">
           <item>
            <property name="text">
             <string>RFC2833</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>信息</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="pagemedia">
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>471</width>
        <height>321</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>媒体设置</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>音频</string>
           </property>
           <widget class="QWidget" name="formLayoutWidget_4">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>30</y>
              <width>211</width>
              <height>71</height>
             </rect>
            </property>
            <layout class="QFormLayout" name="formLayout_5">
             <property name="formAlignment">
              <set>Qt::AlignJustify|Qt::AlignTop</set>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>输入设备</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="audioInputcomboBox"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>输出设备</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QComboBox" name="audioOutputcomboBox"/>
             </item>
            </layout>
           </widget>
           <widget class="QLabel" name="label_12">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>110</y>
              <width>62</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>编解码器</string>
            </property>
           </widget>
           <widget class="QListWidget" name="audioCodeclistWidget">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>130</y>
              <width>101</width>
              <height>151</height>
             </rect>
            </property>
            <item>
             <property name="text">
              <string>PCMU</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>PCMA</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>iLBC</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>gsm</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>G722</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>G7221</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>G729</string>
             </property>
            </item>
           </widget>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>120</x>
              <y>130</y>
              <width>101</width>
              <height>151</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QCheckBox" name="silenceDetectCheckBox">
               <property name="text">
                <string>静音检测</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox">
               <property name="text">
                <string>回声抵消</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>抖动缓冲ms:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="jitterBufferspinBox"/>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>视频</string>
           </property>
           <widget class="QWidget" name="formLayoutWidget_5">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>30</y>
              <width>223</width>
              <height>251</height>
             </rect>
            </property>
            <layout class="QFormLayout" name="formLayout_6">
             <item row="0" column="0">
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>输入设备</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="comboBox"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>大小</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QComboBox" name="comboBox_2">
               <item>
                <property name="text">
                 <string>1080p 1920x1080</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>720p    1280x720</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>VGA     640x480</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>CIF        352x288</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>QVGA    320*240</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>编解码器</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QListWidget" name="audioCodeclistWidget_2">
               <item>
                <property name="text">
                 <string>h263</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>h264</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>webm</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string/>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>360</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>listWidget</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>41</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
